docs/product-user-guide/src/main/asciidoc/business-central-home-con.adoc

```diff
@@ -32,7 +32,7 @@ The Design group contains two pages:
 
 * Projects
 +
-The Projects page enables you to import or  create Business Central projects. This is where you do your process design work. It contains a number of example projects.
+The Projects page enables you to import or  create {CENTRAL} projects. This is where you do your process design work. It contains a number of example projects.
```

Comment:
Double space "..import or  create.." (after "or" and before "create").

Reply:
@michelehaglund